game-server/data/handlers/instance/crucible/EmpyreanCrucibleInstance.java

```diff
@@ -116,788 +109,655 @@ public void onDie(Npc npc) {
+			case 217593 -> 17000;
+			case 217607 -> 17800;
+			case 217599 -> 25900;
+			case 217608 -> 192500;
+			case 217609 -> 204400;
+			default -> 0;
+		};
+		if (points != 0) {
+			addPointsAndSendPacket(points, npc);
 		}
 		switch (npc.getNpcId()) {
-			case 217486:
-				npc.getController().delete();
-				if (stageType == StageType.START_STAGE_1_ROUND_1 && getNpc(217489) == null) {
-					startStage1Round2();
-				}
-				break;
-			case 217489:
+			case 205396, 205405, 217477, 217486, 205399, 205408, 217480, 217489 -> {
 				npc.getController().delete();
-				if (stageType == StageType.START_STAGE_1_ROUND_1 && getNpc(217486) == null) {
+				if (stageType == StageType.START_STAGE_1_ROUND_1 && instance.getNpcs(205396, 205405, 217477, 217486, 205399, 205408, 217480, 217489).isEmpty()) {
 					startStage1Round2();
 				}
-				break;
-			case 217492:
+			}
+			case 217483, 217492 -> {
 				npc.getController().delete();
-				if (stageType == StageType.START_STAGE_1_ROUND_2) {
+				if (stageType == StageType.START_STAGE_1_ROUND_2 && instance.getNpcs(217483, 217492).isEmpty()) {
 					startStage1Round3();
 				}
-				break;
-			case 217487:
+			}
+			case 217478, 217487 -> {
 				npc.getController().delete();
-				if (stageType == StageType.START_STAGE_1_ROUND_3) {
+				if (stageType == StageType.START_STAGE_1_ROUND_3 && instance.getNpcs(217478, 217487).isEmpty()) {
 					startStage1Round4();
 				}
-				break;
-			case 217491:
+			}
+			case 217482, 217491 -> {
 				npc.getController().delete();
-				if (stageType == StageType.START_STAGE_1_ROUND_4) {
+				if (stageType == StageType.START_STAGE_1_ROUND_4 && instance.getNpcs(217482, 217491).isEmpty()) {
 					startStage1Round5();
 				}
-				break;
-			case 217493:
+			}
+			case 217484, 217493 -> {
 				setStage(StageType.PASS_GROUP_STAGE_1, 0);
-				sp(217756, 342.65106f, 357.4013f, 96.09094f, (byte) 0);
-				sp(217756, 349.07376f, 357.5898f, 96.090965f, (byte) 0);
-				sp(217756, 345.69272f, 359.40958f, 96.09094f, (byte) 0);
-				sp(217756, 342.59192f, 353.73386f, 96.090965f, (byte) 0);
-				sp(217756, 342.6043f, 360.8932f, 96.09093f, (byte) 0);
-				sp(217756, 345.69318f, 355.56677f, 96.09094f, (byte) 0);
-				sp(799568, 345.25f, 349.24f, 96.09097f, (byte) 0);
+				sp(217756, 342.65106f, 357.4013f, 96.09094f, (byte) 0); // S1_ResurBox1Dark_55_n
+				sp(217756, 349.07376f, 357.5898f, 96.090965f, (byte) 0); // S1_ResurBox1Dark_55_n
+				sp(217756, 345.69272f, 359.40958f, 96.09094f, (byte) 0); // S1_ResurBox1Dark_55_n
+				sp(217756, 342.59192f, 353.73386f, 96.090965f, (byte) 0); // S1_ResurBox1Dark_55_n
+				sp(217756, 342.6043f, 360.8932f, 96.09093f, (byte) 0); // S1_ResurBox1Dark_55_n
+				sp(217756, 345.69318f, 355.56677f, 96.09094f, (byte) 0); // S1_ResurBox1Dark_55_n
+				sp(799568, 345.25f, 349.24f, 96.09097f, (byte) 0); // IDArena_2nd_MC_Start
 				npc.getController().delete();
-				break;
-			case 217502:
+				sendMsg(SM_SYSTEM_MESSAGE.STR_MSG_INSTANCE_COMPLETE_ROUND_IDARENA());
```

Comment:
This does not compile due to the missing parameter. It also looks like this message should be used in other places too, as well as `STR_MSG_INSTANCE_START_ROUND_IDARENA` and `STR_MSG_INSTANCE_JOIN_ROUND_IDARENA`.